src/transport/DAPIClientTransport/methods/subscribeToBlocks.js

```diff
@@ -9,6 +9,10 @@ module.exports = async function subscribeToBlocks() {
     if (!self.state.block || self.state.block.hash !== chainHash) {
       self.state.block = await self.getBlockByHash(await self.getBestBlockHash());
       self.announce(EVENTS.BLOCK, self.state.block);
+      if (self.state.block && self.state.block.transactions[0].extraPayload.height) {
+        const { height } = self.state.block.transactions[0].extraPayload;
+        self.announce(EVENTS.BLOCKHEIGHT_CHANGED, height);
```

Comment:
The naming is a bit ambiguous. It's not clear that we are talking about chain tip, but not about any other blocks which changed its height for some reason ))

Reply:
Wallet side, how do we get any information about reorg as we only sync based on tx sync from bloomfilter

Reply:
I guess you wil get merkle blocks again. We should test it

Reply:
We decided to implement reorgs handling in a separate PR